test/functional/test_framework/test_framework.py

```diff
@@ -429,7 +433,7 @@ def get_bin_from_version(version, bin_name, bin_default):
                         (version % 10000) // 100,
                         (version % 100) // 1,
                     ),
-                ),
+                ) + suffix,
```

Comment:
not sure if we want to add dead and untested code the the test framework. This can be added when it is actually needed.

Reply:
I need it every time there's a release candidate, but then it never it gets merged...

Reply:
Until we will be testing backwards compatibility for rcs, you can keep it as a commit in your local git and then cherry-pick as needed for your own needs.